docs/how-to/hip_runtime_api.rst

```diff
@@ -0,0 +1,35 @@
+.. meta::
+  :description: HIP runtime API usage
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime API How to,
+
+.. _hip_runtime_api_how-to:
+
+********************************************************************************
+HIP runtime API
+********************************************************************************
+
+The HIP runtime API provides C and C++ functionalities to manage event, stream,
+and memory on GPUs. On AMD ROCm software, the HIP runtime uses :doc:`Common
```

Comment:
The figure shows both PAL and ROCr, so "On AMD ROCm software" should be "On the AMD platform".

Reply:
Fixed.